wazimap_np/templates/homepage.html

```diff
@@ -24,14 +24,14 @@ <h3>
                     <i class="fa fa-book"></i>
                     Find facts
                 </h3>
-                <p>On the beta version of NepalMap, population data is available. Soon more data will be added. </p>
+                <p>On the beta version of NepalMap, demographics, agriculture, education and households data are available.</p>
```

Comment:
<p>On the beta version of NepalMap, demographics, agriculture, educational and households data are available.</p>

Reply:
To me, "households data" does not sound right. I would recommend "household data".

Again, I would suggest all adjectives: " demographic, agricultural, educational and household data".

Reply:
@ishandongol and @ravinepal 

Following up on this comment.

Reply:
@cliftonmcintosh changed to household data.